UpsertedProfileOrchestrator/handler.ts

```diff
@@ -142,18 +136,8 @@ export const getUpsertedProfileOrchestratorHandler = (params: {
           }
         );
 
-        const emailValidationProcessOrchestratorName = pipe(
-          fiscalCode,
-          getIsUserEligibleForNewFeature(
-            cf => params.templateEmailConfig.BETA_USERS.includes(cf),
```

Comment:
We can remove `BETA_USERS` variable as well

Reply:
Solved in [a4b4719](https://github.com/pagopa/io-functions-app/pull/319/commits/a4b47193318862634a4efdda50c132924ae8efad)